frontend/src/pages/GraphPF/styles/styleEdge.tsx

```diff
@@ -31,7 +136,11 @@ const StyleEdgeComponent: React.FC<StyleEdgeProps> = ({ element, ...rest }) => {
     return newData;
   }, [data, detailsLevel]);
 
-  return <BaseEdge element={element} tagClass={tagClass} {...rest} {...passedData} />;
+  return (
+    <g style={{ opacity: opacity }} onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
+      <DefaultEdge className={classes(...cssClasses)} element={element} {...rest} {...passedData} />
```

Comment:
```suggestion
      <DefaultEdge className={classes(...cssClasses)} element={element} tagClass={tagClass} {...rest} {...passedData} />
```
This is needed to support the lock icons on edge tags.